http/preview.go

```diff
@@ -0,0 +1,98 @@
+package http
+
+import (
+	"fmt"
+	"image"
+	"io"
+	"net/http"
+	"net/url"
+	"strings"
+
+	"github.com/disintegration/imaging"
+	"github.com/gorilla/mux"
+
+	"github.com/filebrowser/filebrowser/v2/files"
+)
+
+const (
+	sizeThumb = "thumb"
+	sizeBig   = "big"
+)
+
+type imageProcessor func(src image.Image) (image.Image, error)
+
+var previewHandler = withUser(func(w http.ResponseWriter, r *http.Request, d *data) (int, error) {
+	if !d.user.Perm.Download {
+		return http.StatusAccepted, nil
+	}
+	vars := mux.Vars(r)
+	size := vars["size"]
+	if size != sizeBig && size != sizeThumb {
```

Comment:
I think it should be 
```suggestion
	if size != sizeBig || size != sizeThumb {
```

Reply:
For this condition:
```
if size != sizeBig || size != sizeThumb
```
If the size is "thumb",an error will be returned directly.

Reply:
Oh. My mistake. Newer mind

Reply:
🤣